lib/runner/cli/cli.js

```diff
@@ -176,10 +180,22 @@ class CliRunner {
 
     this.createTestRunner();
     this.setupConcurrency();
+    this.loadTypescriptTranspiler();
 
     return this;
   }
 
+  loadTypescriptTranspiler() {
+    const isTypescriptProject = Utils.fileExistsSync(path.join(process.cwd(), 'tsconfig.json'));
+    if (isTypescriptProject) {
+      Logger.info('Now you can run TS tests directly using Nightwatch.\nFollow this document to learn more.');
```

Comment:
what document?

Reply:
@vishalshah133 Need to work on it, I had removed the line for now